<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>795</width>
                <height>732</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Conway's Game of Life</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <layout class="QVBoxLayout" name="setLayout">
                        <item>
                            <widget class="QLabel" name="label_28">
                                <property name="text">
                                    <string>Life Algorithm</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="algorithmBox">
                                <property name="currentText">
                                    <string/>
                                </property>
                                <property name="currentIndex">
                                    <number>-1</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QGroupBox" name="PowerAlgorithmGroupBox">
                                <layout class="QGridLayout" name="gridLayout_7">
                                    <item row="1" column="2">
                                        <widget class="QComboBox" name="stabilityBornRuleBox"/>
                                    </item>
                                    <item row="3" column="1">
                                        <widget class="QComboBox" name="stabilitySurviveAlgorithmBox"/>
                                    </item>
                                    <item row="1" column="0">
                                        <widget class="QLabel" name="label_31">
                                            <property name="text">
                                                <string>Born</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="1" rowspan="2">
                                        <widget class="QComboBox" name="stabilityBornAlgorithmBox"/>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>Parameter</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="0">
                                        <widget class="QLabel" name="label_33">
                                            <property name="text">
                                                <string>Power Algorithm</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="2">
                                        <widget class="QLabel" name="label_29">
                                            <property name="text">
                                                <string>Rule</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="0" rowspan="2">
                                        <widget class="QLabel" name="label_32">
                                            <property name="text">
                                                <string>Survive</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="3">
                                        <widget class="QLabel" name="label_30">
                                            <property name="text">
                                                <string>Value</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="3" column="2">
                                        <widget class="QComboBox" name="stabilitySurviveRuleBox"/>
                                    </item>
                                    <item row="1" column="3">
                                        <widget class="QDoubleSpinBox" name="stabilityBornBox">
                                            <property name="maximum">
                                                <double>255.000000000000000</double>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="3" column="3">
                                        <widget class="QDoubleSpinBox" name="stabilitySurviveBox">
                                            <property name="maximum">
                                                <double>255.000000000000000</double>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QGroupBox" name="groupBox">
                                <layout class="QGridLayout" name="gridLayout">
                                    <item row="1" column="2">
                                        <widget class="QComboBox" name="comboBox_3"/>
                                    </item>
                                    <item row="0" column="2">
                                        <widget class="QLabel" name="label_4">
                                            <property name="text">
                                                <string>Rule</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="0">
                                        <widget class="QLabel" name="label_6">
                                            <property name="text">
                                                <string>Born</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="2">
                                        <widget class="QComboBox" name="comboBox_4"/>
                                    </item>
                                    <item row="0" column="1">
                                        <widget class="QLabel" name="label_3">
                                            <property name="text">
                                                <string>Parameter</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="3">
                                        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
                                    </item>
                                    <item row="1" column="3">
                                        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                                    </item>
                                    <item row="2" column="1">
                                        <widget class="QComboBox" name="comboBox_2"/>
                                    </item>
                                    <item row="0" column="0">
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>Cell Availiability</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="1">
                                        <widget class="QComboBox" name="comboBox"/>
                                    </item>
                                    <item row="2" column="0">
                                        <widget class="QLabel" name="label_7">
                                            <property name="text">
                                                <string>Survive</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="3">
                                        <widget class="QLabel" name="label_5">
                                            <property name="text">
                                                <string>Value</string>
                                            </property>
                                            <property name="alignment">
                                                <set>Qt::AlignCenter</set>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="0" column="4">
                                        <widget class="QLabel" name="label_8">
                                            <property name="text">
                                                <string>Under Pop</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="1" column="4">
                                        <widget class="QCheckBox" name="checkBox">
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item row="2" column="4">
                                        <widget class="QCheckBox" name="checkBox_2">
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="controlLayout">
                                <item>
                                    <widget class="QPushButton" name="startButton">
                                        <property name="text">
                                            <string>Start</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="stepButton">
                                        <property name="text">
                                            <string>Step</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="stopButton">
                                        <property name="text">
                                            <string>Stop</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="clearButton">
                                        <property name="text">
                                            <string>Clear</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QGridLayout" name="universeSize">
                                <item row="1" column="0">
                                    <widget class="QSpinBox" name="cellSizeX">
                                        <property name="suffix">
                                            <string>cells</string>
                                        </property>
                                        <property name="minimum">
                                            <number>10</number>
                                        </property>
                                        <property name="maximum">
                                            <number>1024</number>
                                        </property>
                                        <property name="value">
                                            <number>50</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label_34">
                                        <property name="text">
                                            <string>Universe Size X</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QLabel" name="label_35">
                                        <property name="text">
                                            <string>Universe Size Y</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QSpinBox" name="cellSizeY">
                                        <property name="suffix">
                                            <string>cells</string>
                                        </property>
                                        <property name="minimum">
                                            <number>10</number>
                                        </property>
                                        <property name="maximum">
                                            <number>1024</number>
                                        </property>
                                        <property name="value">
                                            <number>50</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QLabel" name="label2_4">
                                <property name="text">
                                    <string>Generation interval (in msec)</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QSpinBox" name="iterInterval">
                                <property name="suffix">
                                    <string>ms</string>
                                </property>
                                <property name="minimum">
                                    <number>1</number>
                                </property>
                                <property name="maximum">
                                    <number>100000</number>
                                </property>
                                <property name="singleStep">
                                    <number>1</number>
                                </property>
                                <property name="value">
                                    <number>100</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QGridLayout" name="gridLayout_8">
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label_36">
                                        <property name="text">
                                            <string>Stability</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QLabel" name="powerLimitLabel">
                                        <property name="text">
                                            <string>Power Limit</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="0">
                                    <widget class="QSpinBox" name="stabilityBox">
                                        <property name="maximum">
                                            <number>256</number>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QSpinBox" name="powerLimitBox">
                                        <property name="minimum">
                                            <number>1</number>
                                        </property>
                                        <property name="maximum">
                                            <number>255</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="fileLayout">
                                <item>
                                    <widget class="QPushButton" name="loadButton">
                                        <property name="text">
                                            <string>Load</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="saveButton">
                                        <property name="text">
                                            <string>Save</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QPushButton" name="loadData">
                                <property name="text">
                                    <string>Load Data File</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="colorButton">
                                <property name="text">
                                    <string>Select color for cells</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="rastgele">
                                <property name="text">
                                    <string>Rastgele</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QProgressBar" name="Tamamlanma">
                                <property name="maximum">
                                    <number>100</number>
                                </property>
                                <property name="value">
                                    <number>0</number>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="workDoneLabel">
                                <property name="text">
                                    <string>Work Done</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="elapsedTime">
                                <property name="text">
                                    <string>Elapsed Time</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="remainingTimeLabel">
                                <property name="text">
                                    <string>Remaining Time</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="workSpeedLabel">
                                <property name="text">
                                    <string>Work Speed</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="populationLabel">
                                <property name="text">
                                    <string>Population</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QVBoxLayout" name="gameLayout">
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
